backend/baseclass.pm

```diff
@@ -488,6 +488,8 @@ sub switch_network ($self, $args) { $self->notimplemented }
 
 sub save_memory_dump ($self, $args) { $self->notimplemented }
 
+sub backup_disks ($self, $args) { $self->notimplemented }
```

Comment:
I would avoid the specific term disk and rather use something more generic. How about

```suggestion
sub save_storage ($self, $args) { $self->notimplemented }
```